.github/workflows/update-citations.yaml

```diff
@@ -73,12 +79,15 @@ jobs:
 
       - name: Open pull request with updated citations
         if: |
+          github.event.action != 'closed' &&
           steps.changed.outputs.files_changed == 'true' &&
           inputs.open-pr == true
         uses: peter-evans/create-pull-request@v6
         with:
           branch: citation-update
           title: Periodic citation update
+          body: |
+            To see a live preview of this PR, close (not merge) and reopen it.
```

Comment:
When opening a PR from `on-schedule` citation update, this gives some instructions for how to trigger the PR workflow.